optaplanner-quickstarts/spring-boot-school-timetabling/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.optaplanner.quickstarts</groupId>
+    <artifactId>optaplanner-spring-boot-school-timetabling-quickstart</artifactId>
+    <version>7.38.0-SNAPSHOT</version>
+    <name>OptaPlanner Spring Boot school timetabling quick start</name>
+
+    <properties>
+        <version.org.optaplanner>7.38.0-SNAPSHOT</version.org.optaplanner>
```

Comment:
Needs to change during releases...

Reply:
The quickstart version will be 0.1.0-SNAPSHOT, and it won't have a parent pom, so we can't use ${project.version}..